src/components/ConversationSettings/LinkShareSettings.vue

```diff
@@ -85,14 +85,22 @@
 			<button
 				ref="copyLinkButton"
 				@click.prevent="handleCopyLink">
-				<span class="icon icon-clippy" />{{ t('spreed', 'Copy conversation link') }}
+				<ClipboardTextOutline
+					class="md-icon-alignment"
+					:size="16"
+					decorative
+					title="" />{{ t('spreed', 'Copy conversation link') }}
```

Comment:
```suggestion
					title="" />
				{{ t('spreed', 'Copy conversation link') }}
```

Reply:
eslint didn't complain but I remember scenarios where adding newlines would add extra horizontal spacing.

I'll try it locally...